src/app/file-explorer.js

```diff
@@ -70,9 +67,9 @@ function fileExplorer (appAPI, files) {
   })
 
   var deleteButton = yo`
-    <span class=${css.remove} onclick=${deletePath}>
+    <div class=${css.remove} onclick=${deletePath}>
```

Comment:
a `span` has the same layout properties as a `div` with `dislpay: inline-block;` or `display: inline;` or `display: inline-flex;`.
Maybe this change is/was also styling related :-)

My personal recommendation would be to try to stick to flexbox (until in the future the grid layout becomes widely available)
* http://gridbyexample.com/examples/
* https://css-tricks.com/snippets/css/complete-guide-grid/